gui/storage/forms.py

```diff
@@ -1881,6 +1894,34 @@ def commit(self, path='/mnt/'):
         )
 
 
+class ResilverForm(ModelForm):
+
+    class Meta:
+        fields = '__all__'
+        model = models.Resilver
+        widgets = {
+            'weekday': CheckboxSelectMultiple(
+                choices=choices.WEEKDAYS_CHOICES
+            ),
+        }
+
+    def __init__(self, *args, **kwargs):
+        if len(args) > 0 and isinstance(args[0], QueryDict):
+            new = args[0].copy()
+            fix_time_fields(new, ['begin', 'end'])
+            args = (new,) + args[1:]
```

Comment:
Wouldn't we need an addition of:
```
mchoicefield(self, 'weekday', [1, 2, 3, 4, 5, 6, 7])
```
so that when the user changes it once and then comes back he gets the proper days highlighted in the multiple select.